test/libsolidity/smtCheckerTests/operators/slice_default_end.sol

```diff
@@ -8,13 +8,12 @@ contract C {
 		assert(bytes(b[10:]).length == 20);
 		assert(bytes(b[10:])[0] == 0xff);
 		assert(bytes(b[10:])[5] == 0xff);
-		assert(bytes(b[10:])[19] == 0xaa);
+		// Removed because of Spacer nondeterminism
+//		assert(bytes(b[10:])[19] == 0xaa);
```

Comment:
Can you change this to
```suggestion
		//assert(bytes(b[10:])[19] == 0xaa);
```
I grep for `//assert` to find which ones I should try again when there's a new version

Reply:
Done